parsec/core/gui/central_widget.py

```diff
@@ -240,6 +270,20 @@ def _on_connection_state_changed(self, status, status_exc, allow_systray=True):
                 5000,
             )
 
+    def _on_organization_stats_success(self, job):
+        assert job.is_finished()
+        assert job.status == "ok"
+
+        organization_stats = job.ret
+        self.menu.show_organization_stats(
+            organization_id=self.core.device.organization_id, organization_stats=organization_stats
+        )
+
+    def _on_organization_stats_error(self, job):
+        assert job.is_finished()
+        assert job.status != "ok"
+        show_error(self, _("TEXT_GET_ORGANIZATION_STATS_ERROR"), exception=job.exc)
```

Comment:
Showing an error here is too much. It means that when a user logs in, they will immediately get an error if the backend is not reachable. I would simply set the labels to empty texts.

Reply:
Done